packages/workspace/generators.json

```diff
@@ -72,6 +72,12 @@
       "description": "Moves a project's configuration outside of workspace.json"
     },
 
+    "convert-to-buildable": {
+      "factory": "./src/generators/convert-to-buildable/convert-to-buildable#convertToBuildable",
```

Comment:
It looks like one of these is pointing to the wrong function (I think)

Reply:
You were right